### ISSUE TITLE
Don't inline style.css

### ISSUE THREAD

**Issue by carolinan** [WordPress/twentytwentytwo#205]
**Describe the bug**

The header from style.css is being printed in the source in the editors and the front:

```
<style id='twentytwentytwo-style-inline-css'>
/*
Theme Name: Twenty Twenty-Two
Theme URI: https://github.com/wordpress/twentytwentytwo/
Author: the WordPress team
Author URI: https://wordpress.org/
Description: The WordPress default theme for 2022.
Requires at least: 5.8
Tested up to: 5.8
Requires PHP: 5.6
Version: 0.2
License: GNU General Public License v2 or later
License URI: http://www.gnu.org/licenses/gpl-2.0.html
Text Domain: twentytwentytwo
Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, block-patterns, rtl-language-support, sticky-post, threaded-comments

Twenty Twenty-Two WordPress Theme, (C) 2021 WordPress.org
Twenty Twenty-Two is distributed under the terms of the GNU GPL.
*/
```

</body>
</html>

**To Reproduce**
1. View source.

**Expected behavior**
I expect the file header not to be printed.

**Additional context**
Removing the following from functions.php would solve it, so we need to know if removing it has any side effects.
```
// Add metadata to the CSS stylesheet.
wp_style_add_data( 'twentytwentytwo-style', 'path', get_template_directory() . '/style.css' );
```

**Comment by kjellr** [comment]
We should just enqueue this normally, since it looks like we're likely going to need little bits of CSS for release anyway.

**Comment by jffng** [comment]
Closed by #207
